Modules/Messages/src/Messages.Infrastructure/Persistence/Database/IServiceCollectionExtensions.cs

```diff
@@ -53,8 +53,8 @@ public static void AddDatabase(this IServiceCollection services, DbOptions optio
 
 public class DbOptions
 {
-    public string Provider { get; set; }
-    public string DbConnectionString { get; set; }
+    public string? Provider { get; set; }
+    public string? DbConnectionString { get; set; }
```

Comment:
I think these should be initialized with `null!` since the `DbOptions` are filled like so:
```csharp
    public static void AddDatabase(this IServiceCollection services, Action<DbOptions> setupOptions)
    {
        var options = new DbOptions();
        setupOptions.Invoke(options);

        services.AddDatabase(options);
    }
```

I.e. the object is instanced as an empty object and then passed onto `Invoke` call which fills the data in.